server/routes/bookingJourney/selectAdditionalSupport.test.ts

```diff
@@ -0,0 +1,169 @@
+import type { Express } from 'express'
+import request from 'supertest'
+import * as cheerio from 'cheerio'
+import { SessionData } from 'express-session'
+import { FieldValidationError } from 'express-validator'
+import { appWithAllRoutes, flashProvider } from '../testutils/appSetup'
+import { createMockBookerService, createMockPrisonService } from '../../services/testutils/mocks'
+import TestData from '../testutils/testData'
+import { FlashData } from '../../@types/bapv'
+
+let app: Express
+
+const bookerService = createMockBookerService()
+const prisonService = createMockPrisonService()
+let sessionData: SessionData
+
+const url = '/book-a-visit/select-additional-support'
+
+const bookerReference = TestData.bookerReference().value
+const prisoner = TestData.prisonerInfoDto()
+const prison = TestData.prisonDto()
+const visitors = [
+  TestData.visitorInfoDto({ personId: 1, firstName: 'Visitor', lastName: 'One', dateOfBirth: '1980-02-03' }),
+  TestData.visitorInfoDto({ personId: 2, firstName: 'Visitor', lastName: 'Two', dateOfBirth: '1990-09-03' }),
+  TestData.visitorInfoDto({ personId: 3, firstName: 'Visitor', lastName: 'Three', dateOfBirth: '2024-03-01' }),
+]
+
+afterEach(() => {
+  jest.resetAllMocks()
+  jest.useRealTimers()
+})
+
+describe('Select additional support page', () => {
+  let flashData: FlashData
+
+  describe(`GET ${url}`, () => {
+    beforeEach(() => {
+      flashData = {}
+      flashProvider.mockImplementation((key: keyof FlashData) => flashData[key])
+
+      sessionData = {
+        booker: { reference: bookerReference, prisoners: [prisoner] },
+        bookingJourney: { allVisitors: visitors, selectedVisitors: visitors, prisoner, prison },
+      } as SessionData
+
+      app = appWithAllRoutes({ services: { bookerService, prisonService }, sessionData })
```

Comment:
```suggestion
      app = appWithAllRoutes({})
```
I don't think this page needs any services so no need to pass these - and all the set up for mocking them can then go